sbin/prepareWorkspace.sh

```diff
@@ -164,6 +166,27 @@ checkingAndDownloadingAlsa()
   fi
 }
 
+checkFingerPrint() {
+  local sigFile="$1"
+  local fileName="$2"
+  local publicKey="$3"
+  local expectedFingerPrint="$4"
+
+  rm /tmp/public_key.gpg || true
+  gpg --output /tmp/public_key.gpg --dearmor "${SCRIPT_DIR}/sig_check/${publicKey}.asc"
+
+  gpg -v --no-default-keyring --keyring "/tmp/public_key.gpg" --verify $sigFile $fileName
+
+  local verify=$(gpg -v --no-default-keyring --keyring "/tmp/public_key.gpg" --verify $sigFile $fileName 2>&1)
```

Comment:
Effectively executing the same command as above?

Reply:
Yeah, first one is so we can see the output in the log